src/Microsoft.Health.Fhir.Core/Features/Operations/IIntegrationDataStoreClientInitilizer.cs

```diff
@@ -3,27 +3,55 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using System.Threading;
+using System;
 using System.Threading.Tasks;
 using Microsoft.Health.Fhir.Core.Configs;
 
 namespace Microsoft.Health.Fhir.Core.Features.Operations
 {
-    public interface IIntegrationDataStoreClientInitilizer<T>
+    public interface IIntegrationDataStoreClientInitilizer<TBlobServiceClient, TBlockBlobClient, TBlobClient>
```

Comment:
nit: Fix typo: 
```suggestion
            IIntegrationDataStoreClientInitializer<BlobServiceClient, BlockBlobClient, BlobClient> initializer = GetClientInitializer();
```

Reply:
Small refactor, but from what I can see, this interface could be moved to `src/Microsoft.Health.Fhir.Azure`. This would then mean the blob types can be referenced and it wouldn't need the generics. Thoughts?

Reply:
The interface can be moved to src/Microsoft.Health.Fhir.Azure. The only reason I see is that the Export counterpart follows the same pattern (or vice versa) having the interface in the Core folder and the implementation in the Azure folder. If that doesn't matter to you, I will move the interface for import to the Azure folder and make it non-generics.

Reply:
I'd leave the other interfaces, but this one isn't referenced by anything in Core, so it seems like it could just reside closer to its implementations. This interface seems overly complicated, without direct knowledge of the actual blob client implementations. I think moving it would help simplify

Reply:
I moved the interface to Azure, made it non-generics, and fixed the typo. I left some comments as questions to everyone. Please review them and leave any comments you have. Thank you.